sys/net/network_layer/ipv6/addr/ipv6_addr_from_str.c

```diff
@@ -163,6 +163,31 @@ ipv6_addr_t *ipv6_addr_from_str(ipv6_addr_t *result, const char *addr)
     return ipv6_addr_from_buf(result, addr, strlen(addr));
 }
 
+int ipv6_prefix_from_str(ipv6_addr_t *result, const char *addr)
+{
+    size_t str_len;
+    int pfx_len;
+    const char *separator;
+
+    if ((result == NULL) || (addr == NULL)) {
+        return -1;
+    }
+
+    if ((separator = strrchr(addr, '/')) == NULL) {
+        str_len = strlen(addr);
+        pfx_len = 128;
```

Comment:
So any address is just interpreted as prefix, even if a prefix marker is missing? Seems fishy ^^...

Reply:
But isn't a single address a /128?

Reply:
Semantics... an address is an address, not a prefix. I'd prefer to have the user forced to configure a prefix, and not do some convenience magic if they try to configure an address, just to be sure they do what they meant to do.

Reply:
Ok, dropped it.